### PR TITLE
Fwmt 992 outcome non valid household derelict

### PR DESCRIPTION
**Pull request by JacobHarrisonQA**

### CODE REVIEW

File: src/main/java/uk/gov/ons/census/fwmt/outcomeservice/service/CometTranslationService.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package uk.gov.ons.census.fwmt.outcomeservice.service;
+
+import uk.gov.ons.census.fwmt.common.error.GatewayException;
+import uk.gov.ons.census.fwmt.outcomeservice.data.dto.comet.HouseholdOutcome;
+
+public interface CometTranslationService {
+
+  void transformCometPayload(HouseholdOutcome householdOutcome) throws GatewayException;
```
Comment (spdiaz1000) [inline src/main/java/uk/gov/ons/census/fwmt/outcomeservice/service/CometTranslationService.java]: rename to CreateHouseHoldOutComeEvent

File: src/main/java/uk/gov/ons/census/fwmt/outcomeservice/service/CometTranslationService.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package uk.gov.ons.census.fwmt.outcomeservice.service;
+
+import uk.gov.ons.census.fwmt.common.error.GatewayException;
+import uk.gov.ons.census.fwmt.outcomeservice.data.dto.comet.HouseholdOutcome;
+
+public interface CometTranslationService {
```
Comment (spdiaz1000) [inline src/main/java/uk/gov/ons/census/fwmt/outcomeservice/service/CometTranslationService.java]: Rename to OutcomeService

File: src/main/java/uk/gov/ons/census/fwmt/outcomeservice/service/impl/CometTranslationServiceImpl.java [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    buildOutcome(householdOutcome, outcomeEvent);
+
+    Event event = new Event();
+    event.setTransactionId("c45de4dc-3c3b-11e9-b210-d663bd873d93");
+
+    outcomeEvent.setEvent(event);
+    outcomeService.sendOutcome(outcomeEvent);
+  }
+
+  private void buildOutcome(HouseholdOutcome householdOutcome,
```
Comment (spdiaz1000) [inline src/main/java/uk/gov/ons/census/fwmt/outcomeservice/service/impl/CometTranslationServiceImpl.java]: Leave as is for now but...
In the back of my mind I have the idea how we had different convertors in Jobservice. It was a lot neater, but this has the extra complication of secondary reasons

File: src/main/java/uk/gov/ons/census/fwmt/outcomeservice/data/dto/comet/HouseholdOutcome.java [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  private String secondaryOutcome;
+  @JsonProperty("outcomeNote")
+  private String outcomeNote;
+  @JsonProperty("ceDetails")
+  private CeDetails ceDetails;
+  @JsonProperty("fulfillmentRequests")
+  private List<FulfillmentRequest> fulfillmentRequests = null;
+  @JsonIgnore
+  private Map<String, Object> additionalProperties = new HashMap<>();
+
```
[the remark this answers is not in the document]
Comment (JacobHarrisonQA) [reply src/main/java/uk/gov/ons/census/fwmt/outcomeservice/data/dto/comet/HouseholdOutcome.java]: good point will investigate

**Pull request merged**